fuel-core/src/cli/run/p2p.rs

```diff
@@ -113,6 +113,10 @@ pub struct P2pArgs {
     /// Choose how long RequestResponse protocol connections will live if idle
     #[clap(long = "connection_keep_alive", default_value = "20")]
     pub connection_keep_alive: u64,
+
+    /// Enable p2p metrics or not
+    #[clap(long = "metrics", default_value = "false")]
```

Comment:
it might be better to just have a global metrics flag, otherwise we'll need to enable metrics on a per service basis:
`--p2p-metrics --txpool-metrics --blockproducer-metrics` etc.

```suggestion
    #[clap(long = "p2p-metrics", default_value = "false")]
```

Reply:
this name conflicts with the main metrics flag. Repro:

```
cargo run -p fuel-core --features p2p -- run --help

thread 'main' panicked at 'Command run: Argument names must be unique, but 'metrics' is in use by more than one argument or group',
```